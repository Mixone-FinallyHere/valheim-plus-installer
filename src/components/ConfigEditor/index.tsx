import { makeStyles } from "@material-ui/core/styles";
import React, { useEffect, useState } from "react";
import Card from "@material-ui/core/Card";
import Typography from "@material-ui/core/Typography";

interface ConfigEditorProps {
  state: IState;
  stateDispatch: StateDispatch;
}

const useStyles = makeStyles((theme) => ({
  card: {
    display: "flex",
    flexFlow: "row wrap",
    justifyContent: "center",
    width: 500,
    margin: theme.spacing(1),
    marginLeft: "auto",
    marginRight: "auto",
    padding: theme.spacing(4),
  },
  quitMessage: {
    marginTop: 15,
  },
}));

function ConfigEditor(props: ConfigEditorProps): React.ReactElement {
  const { state, stateDispatch } = props;
  const classes = useStyles();

  return (
    <Card className={classes.card}>
      <Typography variant="h4">CFG Editor Coming Soon</Typography>
      <Typography className={classes.quitMessage}>You can now safely close this application</Typography>
    </Card>
  );
}

export default ConfigEditor;
