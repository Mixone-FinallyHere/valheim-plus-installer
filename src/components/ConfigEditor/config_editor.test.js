import React from "react";
import ConfigEditor from ".";
import { render } from "@testing-library/react";

it("renders without crashing", () => {
  const { asFragment } = render(
    <ConfigEditor />
  );

  expect(asFragment()).toMatchSnapshot();
});
