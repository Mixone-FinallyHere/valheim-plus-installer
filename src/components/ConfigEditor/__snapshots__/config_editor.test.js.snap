// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="MuiPaper-root MuiCard-root makeStyles-card-1 MuiPaper-elevation1 MuiPaper-rounded"
  >
    <h4
      class="MuiTypography-root MuiTypography-h4"
    >
      CFG Editor Coming Soon
    </h4>
    <p
      class="MuiTypography-root makeStyles-quitMessage-2 MuiTypography-body1"
    >
      You can now safely close this application
    </p>
  </div>
</DocumentFragment>
`;
